### ISSUE TITLE
Update sdk version

### ISSUE THREAD

**Issue by DCrow** [apptreesoftware/flutter_map#93]
Trying to run `flutter run` in example.
Getting
```
Running "flutter packages get" in flutter_map_example...
The current Dart SDK version is 2.1.0-dev.1.0.flutter-ccb16f7282.

Because flutter_map_example depends on flutter_map from path which requires SDK version >=1.19.0 <2.0.0, version solving failed.

pub get failed (1)
```

Is it possible to update pubspec.yaml to allow new SDK ?

**Comment by devjv** [comment]
👍 
Solved for now by editing the pubspec.yaml of a locally cached version or forking this and doing the same.
[Info how to use local or Github version of a Dart package here.](https://flutter.io/using-packages/#dependencies-on-unpublished-packages)

**Comment by johnpryan** [comment]
fixed in version 0.1.1
